From: John Zupin <johnz@lunarg.com>
Date: Wed, 17 Jan 2024 14:07:54 -0700
Subject: patch: remove installing the static library

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a06517f..5e9ec60 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -99,7 +99,7 @@ if(VOLK_INSTALL)
   install(FILES volk.h volk.c DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
   # Install library target and add it and any dependencies to export set.
-  install(TARGETS volk volk_headers
+  install(TARGETS volk_headers
     EXPORT volk-targets
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
